rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/monitoring/charts/CpusUsageLineChart.java

```diff
@@ -66,26 +61,25 @@ public void processResult(String result) {
             String timeStamp = DateTimeFormat.getFormat(PredefinedFormat.HOUR24_MINUTE)
                                              .format(new Date(System.currentTimeMillis()));
 
-            addRow();
-            loadTable.setValue(loadTable.getNumberOfRows() - 1, 0, timeStamp);
+            addXLabel(timeStamp);
 
-            boolean initColumns = super.initColumns();
-            int colIndex = 1;
+            //            boolean initColumns = super.initColumns();
```

Comment:
Remove unused code ?

Reply:
I will remove in final PR. Work is not finished yet.